setup.py

```diff
@@ -23,23 +23,30 @@
         # https://github.com/Unidata/netcdf4-python/issues/1179
         netCDF4_requirement = "netCDF4>=1.6.2"
 
+necessary_require = [
+    "numpy>=1.0",
+    "scipy>=1.0",
+    "matplotlib>=3.0",
+    netCDF4_requirement,
+    "requests",
+    "pytz",
+    "simplekml",
+]
+
+env_analysis_require = [
+    "timezonefinder",
+    "windrose>=1.6.8",
+    "ipywidgets>=7.6.3",
+    "jsonpickle",
+]
+
```

Comment:
Fantastic variable usage here, also it is much easier to locate the configuration for the netCFD4 package. Well done.